src/pages/versions.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import Layout from "@theme/Layout";
+import Link from "@docusaurus/Link";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import versions from "@site/versions.json";
+import VersionsArchived from "@site/versionsArchived.json";
+
+// importing container component
+import Container from "@site/src/components/v1/container";
+import "@site/src/components/v1/cascading.css";
+
+export default function VersionsPage() {
+  const { siteConfig } = useDocusaurusContext();
+  const latestVersion = versions[0];
+  return (
+    <Layout>
+      <div className="wrapperV1">
+        <Container className="docsContainer">
+          <div></div>
```

Comment:
Let's wrap the whole content with `<main className="container margin-vert--lg"></main>`. The `wrapperV1` and `Container` won't be needed anymore.